.github/workflows/static-analysis-src.yml

```diff
@@ -1,4 +1,4 @@
-name: static analysis api
+name: static analysis src
```

Comment:
Btw, ignore the fact that this seems like I renamed `api` to `src` and visa versa. It's because git can't resolve this properly in the diff. The files are still correct.